src/main/java/org/wildfly/security/password/impl/OneTimePasswordAlgorithmParametersSpiImpl.java

```diff
@@ -54,7 +54,7 @@ protected void engineEncode(final ASN1Encoder encoder, final OneTimePasswordAlgo
     protected OneTimePasswordAlgorithmSpec engineDecode(final ASN1Decoder decoder) {
         decoder.startSequence();
         final String algorithm = decoder.decodeOctetStringAsString();
-        final byte[] seed = decoder.decodeOctetString();
+        final String seed = decoder.decodeOctetStringAsString();
```

Comment:
@fjuma I though it was unnecessary because we are encoding a String, if we have to do that, then we have to do the same with algorithm, WDYT?